### PR TITLE
Makes basic tools a starting node instead of having to research them and reverts #41092

### LINKED ISSUE #41091: Advanced tools node doesn't require regular tools
This is inconsistent and doesn't make much sense for jaws and such to be researched while lathes can't print out basic tools...

### LINKED ISSUE #41213: Autolathe no longer has any of the usual engineering/medical tools regardless of research
Issue reported from  Round ID: 96253 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])
Reporting client version: 512
#41029: '[SUPER DUPER EXTRA READY] Hilbert's Hotel' by TheDreamweaver at commit 47b488e326

@Cyberboss @ShizCalev this is your fault because you two are the ones who were touching techwebs, though not 41092 as that's not on the server yet

### PR DESCRIPTION
**Pull request by Barhandar**
Closes #41091
Closes #41213
Reverts #41092 because node change is unnecessary with this + autolathe tools needing a design disk upload and hence cooperation from either science or engineering was _very bad_

:cl: Barhandar
balance: Basic tools are now a starting node and do not need researching.
tweak: Autolathe tools can now be printed again without needing a design disk.
/:cl:

Because it's bloody stupid that you have to research something you can find strewn over the station, or ask two departments that can print tools for themselves freely to get a disk for your lathe.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
If something req's this node it should cost more to compensate.

If you're removing the node you'll need to `closes #41092` as well

**Comment by Barhandar** [comment]
Nothing requires this node at the moment.
The node _technically_ isn't being removed (that would be adding the stuff it makes to base), it's being made starting node which is unlocked by default. Though I guess it does make sense to close the other PR because it's meaningless.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Undo the changes from #41092

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -491,10 +491,8 @@
 	id = "basic_tools"
```
Comment (ShizCalev) [inline code/modules/research/techweb/all_nodes.dm]: just noticed, move this to the top of the file with the rest of the starting nodes, then all good to merge.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**